build-debezium.sh

```diff
@@ -42,9 +42,13 @@ build_docker_image () {
         if [ "$PUSH_IMAGES" == "true" ]; then
             echo "Pushing the stream image into the registry"
             docker push "debezium/${IMAGE_NAME}:${IMAGE_TAG}"
+            docker tag "debezium/${IMAGE_NAME}:${IMAGE_TAG}" "quay.io/debezium/${IMAGE_NAME}:${IMAGE_TAG}"
+            docker push "quay.io/debezium/${IMAGE_NAME}:${IMAGE_TAG}"
             if [ "$DEBEZIUM_VERSION" == "$LATEST_STREAM" ]; then
                 echo "Pushing the latest image into the registry"
                 docker push "debezium/${IMAGE_NAME}:latest"
+                docker tag "debezium/${IMAGE_NAME}:${IMAGE_TAG}" "quay.io/debezium/${IMAGE_NAME}:${IMAGE_TAG}"
```

Comment:
@uidoyen This change is not correct. There was `latest` tag used originally and what was expected is that you will add `latest` tag for `quay.io` otherwise the push that was there before this correction would not work.

Reply:
@jpechane updated with `latest` tag